### PR TITLE
Update url-rewriting.md

### LINKED ISSUE #21312: url rewrite doc updates
[Enter feedback here]
In the section on **[Unsupported features](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/url-rewriting?view=aspnetcore-5.0#unsupported-features)** , is that still accurate for .net 5? It references asp.net core 2. 

I'm also interested in performance considerations. I haven't seen anything on the web for this or in this article. I just added a SO question to see if the community has input. middleware vs. IIS rewrite module is what I'm trying to figure out. Are there reasons to pick one over the other? I'm assuming the request is processed differently in the pipeline using the IIS module vs. code middleware. 

Thanks for the existing detailed documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7f5f7c61-a9fa-bd34-001d-217477874bb6
* Version Independent ID: 9c48da92-af90-3fc1-fde0-c3f74fa88510
* Content: [URL Rewriting Middleware in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/url-rewriting?view=aspnetcore-5.0)
* Content Source: [aspnetcore/fundamentals/url-rewriting.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/url-rewriting.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #21312

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@mikaelm12 30 second review please.

**Review by mikaelm12 — APPROVED** [review]

**Comment by mikaelm12** [comment]
Hey Rick, I switched teams and it looks like I don't have the right access anymore to approve PRs here. Maybe @BrennanConroy or @jkotalik can unblock you here

**Pull request merged**
